react/components/EXPERIMENTAL_Table/BulkActions/Actions.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react'
+import csx from 'classnames'
+
+import Button from '../../Button'
+import ActionMenu from '../../ActionMenu'
+import { MenuAction } from '../Toolbar/PopoverMenu'
+import { ORDER_CLASSNAMES } from '../constants'
+
+const ACTION_MARGIN = 'mr4'
+
+export default function Actions({ children }) {
+  const className = csx('flex flex-row', ORDER_CLASSNAMES.BULK_CHILD.ACTIONS)
+  return <div className={className}>{children}</div>
+}
+
+Actions.Primary = Primary
+Actions.Secondary = Secondary
+
+function Primary({ label, onClick }: PrimaryProps) {
+  const className = csx(
+    ACTION_MARGIN,
+    ORDER_CLASSNAMES.BULK_CHILD.ACTIONS_CHILD.PRIMARY
+  )
+  return (
+    <div className={className}>
+      <Button variation="secondary" size="small" onClick={onClick}>
+        {label}
+      </Button>
+    </div>
+  )
+}
+
+function Secondary({ label, actions, onActionClick }: SecondaryProps) {
+  const className = csx(
+    ACTION_MARGIN,
+    ORDER_CLASSNAMES.BULK_CHILD.ACTIONS_CHILD.SECONDARY
+  )
+  return (
+    <div className={className}>
+      <ActionMenu
+        label={label}
+        buttonProps={{ variation: 'secondary', size: 'small' }}
+        options={actions.map(el => ({
+          label: el.label,
+          onClick: () => onActionClick(el),
+        }))}
+      />
+    </div>
+  )
+}
+
+type PrimaryProps = {
+  label: string
+  onClick: () => void
+}
+
+type SecondaryProps = {
+  label: string
+  actions: Array<MenuAction>
+  onActionClick: (el: MenuAction) => void
+}
```

Comment:
I would just change the order of definitions in the file. I think it would be more readable. :thinking: 
(type definitions before functions definitions)

Reply:
I prefer the definitions on the bottom because I like to read directly the component content instead of scrolling through a bunch of type definitions. The full definition doesn't matter most of the time, especially if you just opened the file and did not saw in which context are these types beeing used. `OBS`: If you use VS Code you can preview the type by just hovering it with the mouse